### PR TITLE
Examples gallery

### PR DESCRIPTION
**Pull request by shilman**
Issue: #45 

## What I did

localhost_8000_examples_

- first iteration complete
- suggestions welcome
- more examples welcome

### CODE REVIEW

**Comment by aaronmcadam** [comment]
Here's a few example storybooks I learned from:

* https://github.com/diegohaz/arc
* https://react-move.js.org/#/story/readme-documentation
* https://github.com/sm-react/react-theming

File: pages/examples/_examples.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  description: A collection of Buffer UI React components.
+  source: https://github.com/bufferapp/buffer-components
+  demo: https://bufferapp.github.io/buffer-components/
+  site: https://buffer.com
+lonelyplanet:
+  thumbnail: ./thumbnails/lonelyplanet.jpg
+  title: Lonely Planet
+  description: All the tools you need to build the Lonely Planet UI experience.
+  source: https://github.com/lonelyplanet/backpack-ui
+  site: https://www.lonelyplanet.com/
```
Comment (aaronmcadam) [inline pages/examples/_examples.yml]: `demo: https://lonelyplanet.github.io/backpack-ui`

**Comment by ndelangen** [comment]
Adding company logos to the tiles would be nice, especially if it's a well known brand.

**Comment by usulpro** [comment]
Do you have this one in the list?

**deleted for privacy**

**Comment by shilman** [comment]
cc @sachag

**Review by ndelangen — APPROVED** [review]

**Comment by shilman** [comment]
OK merging v0 of this. Next version will contain logos, better design, etc.

**Pull request merged**